kubectl-plugin/src/resources/volume.rs

```diff
@@ -111,3 +115,40 @@ impl Scale for Volume {
         }
     }
 }
+
+#[async_trait(?Send)]
+impl ReplicaTopology for Volume {
+    type ID = VolumeId;
+    async fn topology(id: &Self::ID, output: &OutputFormat) {
+        match RestClient::client().volumes_api().get_volume(id).await {
+            Ok(volume) => {
+                // Print table, json or yaml based on output format.
+                utils::print_table(output, volume.state.replica_topology);
+            }
+            Err(e) => {
+                println!("Failed to get volume {}. Error {}", id, e)
```

Comment:
on an error do we exit the process with an error code?

Reply:
Not currently, we only print the error. I will raise a Jira for this.